mappings/net/minecraft/client/render/entity/model/HoglinEntityModel.mapping

```diff
@@ -1 +1,5 @@
 CLASS net/minecraft/class_4791 net/minecraft/client/render/entity/model/HoglinEntityModel
+	FIELD field_22227 head Lnet/minecraft/class_630;
+	FIELD field_22228 rightEar Lnet/minecraft/class_630;
+	FIELD field_22229 leftEar Lnet/minecraft/class_630;
```

Comment:
These are from the perspective of the entity, right?

Reply:
yes, but i am not very sure (need rune to confirm with his cool mod)

Reply:
Can't do that right now, maybe on Sunday. 

Here's the mod, if you want to do it (or just need it for something else in the future): https://github.com/Runemoro/model-part-namer